### PR TITLE
Fix a few small API bugs

### PR DESCRIPTION
**Pull request by steve-gregory**
## Description
These changes were requested by @jchansen who found them during front-end development.

Included:
- Allow 'search' of tags API
- Updated queryset for Image Bookmark API

## Checklist before merging Pull Requests
- [ ] New test(s) included to reproduce the bug/verify the feature
- [ ] Reviewed and approved by at least one other contributor.

### CODE REVIEW

File: core/models/application.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def shared_with_user(user, is_leader=None):
         elif is_leader == True:
             project_query &= Q(projects__owner__memberships__is_leader=True)
         membership_query = Q(created_by__memberships__group__user=user)
-        return Application.objects.filter(membership_query | project_query | ownership_query).distinct()
+        return Application.objects.filter(query.only_current_apps() | membership_query | project_query | ownership_query).distinct()
```
Comment (cdosborn) [inline core/models/application.py]: I think you want an AND here. (A | B | C) & current

Comment (cdosborn) [reply core/models/application.py]: Like line 125 above

Comment (steve-gregory) [reply core/models/application.py]: @cdosborn good point, i'll write up a test-case before requesting review to ensure we get the expected results.

**Comment by julianpistorius** [comment]
@cdosborn Is this one ready?

**Comment by cdosborn** [comment]
Had it on my todo to build and run.

**Pull request closed without merging**